dev/com.ibm.ws.jpa.container.ormdiagnostics_fat/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.target: 1.8
+javac.source: 1.8
+
+src: \
+	fat/src, \
+	test-applications/jpasimple/src
+
+fat.project: true
+
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
+-buildpath: \
```

Comment:
This list in the build path seems excessive.  Do you really need cdi, ejb, jms and validation?

Reply:
For the simple JPA app that I use, nah, I don't need all those.

Reply:
Removed, and pushed an update commit including all of your recommended changes.